session.go

```diff
@@ -117,8 +117,11 @@ type session struct {
 	version        protocol.VersionNumber
 	config         *Config
 
-	conn      connection
-	sendQueue *sendQueue
+	conn connection
+
+	sendQueue          *sendQueue
+	sendQueueBlocked   bool
```

Comment:
Can we avoid this bool by exposing the sendQueue's blocked bool?

Reply:
I don't think that works, since that bool could change before the run loop is started the next time.